### PR TITLE
issue 373.3: use TRI_vocbase_t& for Upgrade tasks, remove redundant checks for null TRI_vocbase_t

### PR DESCRIPTION
**Pull request by vasiliy-arangodb**

### CODE REVIEW

File: arangod/VocBase/Methods/UpgradeTasks.cpp [hunk 1]
```diff
@@ -307,50 +354,93 @@ bool UpgradeTasks::insertRedirections(TRI_vocbase_t* vocbase,
...
+  );
+  ::createIndex(
+    &vocbase,
+    "_jobs",
+    Index::TRI_IDX_TYPE_SKIPLIST_INDEX,
+    {"status", "queue", "delayUntil"},
+    /*unique*/ true,
+    /*sparse*/ true
+  );
+
```
Comment (graetzer) [inline arangod/VocBase/Methods/UpgradeTasks.cpp]: Are you using clang-format for this ?

Comment (vasiliy-arangodb) [reply arangod/VocBase/Methods/UpgradeTasks.cpp]: "clang-format"? I didn't change the arguments, only changed the pointer to a reference.

Comment (graetzer) [reply arangod/VocBase/Methods/UpgradeTasks.cpp]: I know but you are reformatting a lot of stuff, for which we generally try to use the automated tool `clang-format`

Comment (vasiliy-arangodb) [reply arangod/VocBase/Methods/UpgradeTasks.cpp]: I format by hand for readability

**Pull request merged**